src/mpc-hc/mpc-hc.rc

```diff
@@ -1542,6 +1542,9 @@ BEGIN
             MENUITEM "&200%",                       ID_VIEW_ZOOM_200
             MENUITEM "Auto &Fit",                   ID_VIEW_ZOOM_AUTOFIT
             MENUITEM "Auto Fit (&Larger Only)",     ID_VIEW_ZOOM_AUTOFIT_LARGER
+            MENUITEM SEPARATOR
```

Comment:
Adding menu entries seems pretty pointless to me for this functionality. Nobody is going to use that. Better to remove them to keep menus clean.